dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/monitor/AdvancedRumMonitor.kt

```diff
@@ -26,6 +26,8 @@ internal interface AdvancedRumMonitor : RumMonitor {
 
     fun addResourceTiming(key: String, timing: ResourceTiming)
 
+    fun addLongTask(durationNs: Long, target: String)
```

Comment:
you do not want to expose this to Public API ?

Reply:
No, there's no point in making this public for now (there's no manual Long Task api in browser either)